src/main/java/io/opentracing/contrib/concurrent/TracedCallable.java

```diff
@@ -11,20 +12,24 @@
 public class TracedCallable<V> implements Callable<V> {
 
   private final Callable<V> delegate;
-  private final Continuation continuation;
+  private final Span continuation;
+  private final Tracer tracer;
 
-  public TracedCallable(Callable<V> delegate, ActiveSpan activeSpan) {
+  public TracedCallable(Callable<V> delegate, Tracer tracer) {
```

Comment:
Is there a way to keep a backwards compatible version available, perhaps with a `@Deprecated` tag?

Reply:
Backward compatibility would be impossible - the `ActiveSpan` class no longer exists.

Reply:
Good point :)